packages/sdk/contractkit/src/wrappers/EpochManager.test.ts

```diff
@@ -0,0 +1,78 @@
+import { newEpochManager } from '@celo/abis-12/web3/EpochManager'
+import { testWithAnvilL2 } from '@celo/dev-utils/lib/anvil-test'
+import { timeTravel } from '@celo/dev-utils/lib/ganache-test'
+import Web3 from 'web3'
+import { newKitFromWeb3 } from '../kit'
+
+process.env.NO_SYNCCHECK = 'true'
```

Comment:
is this actually right?

Reply:
i mean in this case we are interacting with chain so we would want node to be synced

Reply:
yes, we have it for all the tests interacting with the CLI